org.mixedrealitytoolkit.standardassets/Icons/MRTK_FluentIconSet.asset

```diff
@@ -298,5 +298,5 @@ MonoBehaviour:
       value: 63703
     - key: Icon 142
       value: 63742
-  iconFontAsset: {fileID: 11400000, guid: 9b0d0ee11ff70b04d901a29b519cbaa0, type: 2}
+  iconFontAsset: {fileID: 11400000, guid: 533bdd8d5c92b52448ee2ecf7bd828a4, type: 2}
```

Comment:
Should still refence "9b0d0ee11ff70b04d901a29b519cbaa0"....so users of this asset don't get unexpected glyphs

9b0d0ee11ff70b04d901a29b519cbaa0 = MRTK_FluentIconSet SDF.asset
533bdd8d5c92b52448ee2ecf7bd828a4 = selawksb SDF.asset (or now known as Selawik-Semibold-MRTKIcons SDF.asset)

How about we create a new "Font Icon Set" in the "Fonts" folder.  The new "Font Icon Set" would reference the reference "533bdd8d5c92b52448ee2ecf7bd828a4".   Then the button prefabs should reference the new "Font Icon Set"

Reply:
Done with commits https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/pull/700/commits/c6c51b5dd2eda1872988e58feee95cb060922722 and https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/pull/700/commits/278ca7b25aedf2492785675eed2466e2a7cb3bbd, resolving this.